src/components/layout-editor-tool/layout-editor-tool-bar/layout-editor-tool-bar.tsx

```diff
@@ -43,18 +49,26 @@ export class LayoutEditorToolBar {
     this.parentControl = getBreadcrumb(this.control)[0];
   }
 
+  private refreshButtonsStatus() {
+    if (this.parentControl) {
+      this.itemButtonSelect.classList.remove("hidden");
+    } else {
+      this.itemButtonSelect.classList.add("hidden");
+    }
+  }
+
   render() {
     return [
       <div class="bar">
         <gx-le-tool-identity class="identity" control={this.control} />
         <ul class="buttons">
-          <li>
-            <button class="info" />
+          <li class="info hidden" ref={el => (this.itemButtonInfo = el)}>
```

Comment:
Related to previous comment, you could do something like this:

```suggestion
          <li class="info" hidden ref={el => (this.itemButtonInfo = el)}>
```